DSharpPlus/Entities/Integration/DiscordIntegrationAccount.cs

```diff
@@ -28,8 +28,14 @@ namespace DSharpPlus.Entities
     /// <summary>
     /// Represents a Discord integration account.
     /// </summary>
-    public class DiscordIntegrationAccount : SnowflakeObject
+    public class DiscordIntegrationAccount
     {
+        /// <summary>
+        /// Gets the ID of this object. This ID is a string and not a snowflake ulong.
```

Comment:
```suggestion
        /// Gets the ID of the account.
```

Reply:
![794514391500193812](https://user-images.githubusercontent.com/33007665/129483392-4bb97b2a-133f-4059-8cc9-8a9b57aa2887.gif)

Reply:
There's no need to explicitly say that it's not a snowflake, that's inferred from the fact that it's a `string`

Reply:
![664057778482184195](https://user-images.githubusercontent.com/33007665/129483492-b4578864-9623-4b1e-a34a-cf103b1f81ef.gif)